.github/workflows/adobe-activation.yml

```diff
@@ -0,0 +1,39 @@
+name: Update Adobe Activation List
+
+on:
+  schedule:
+    - cron: '0 3 * * *' # Run every day at 3:00 UTC
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+  update-adobe-activation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Download pihole.txt
+        run: |
+          curl -sSL https://raw.githubusercontent.com/ignaciocastro/a-dove-is-dumb/main/pihole.txt -o new-pihole.txt
```

Comment:
By the time I checked this source, it was updated within 24hrs, which is good.
But I am not sure this source will get long enough maintenance, that is a NO-GO to me to use a third-party source.

BTW, you have to make sure that we will not violate their open-source license. You might just forget to mention it in this PR.